outline/functions.md

```diff
@@ -12,171 +12,83 @@ http://clojurebridge.github.io/curriculum/outline/functions.html
 
 {% endcomment %}
 
-<section>
+<section ng-controller="NarrativeController">
```

Comment:
I actually don't know. I just saw it all over the newer slides so I replicated it.